descheduler.yaml

```diff
@@ -0,0 +1,42 @@
+package:
+  name: descheduler
+  version: 0.30.1
+  epoch: 0
+  description: Descheduler for Kubernetes
+  copyright:
+    - license: Apache-2.0
+
+environment:
```

Comment:
you don't need to specify this brother.

Reply:
I included looking here https://github.com/kubernetes-sigs/descheduler/blob/0f1890e5cd2782c580951b20b04f40a5ed0f6642/Makefile#L62 but I agree on removing it.

Reply:
go/build pipeline should be taking care of it.